### PR TITLE
Update and move ENS Initial Registrar spec to new EIP format

### PR DESCRIPTION
**Pull request by maurelian**
@Arachnid @alexvandesande I've update this to reflect the current design, but would appreciate a quick review to see if I've missed anything. Hopefully moving this to the new EIP format will remove me as a bottle neck for updating the spec. 

Similar to #629, this is already implemented and deployed.

### CODE REVIEW

File: EIPS/eip-ens-initial-registrar.md [hunk 1]
```diff
@@ -0,0 +1,244 @@
+```
+EIP: Draft
```
Comment (Arachnid) [inline EIPS/eip-ens-initial-registrar.md]: You can set this to 162.

File: EIPS/eip-ens-initial-registrar.md [hunk 2]
```diff
@@ -0,0 +1,244 @@
...
+Title: Initial ENS Hash Registrar
+Author: Maurelian and Nick Johnson
+Status: Draft
```
Comment (Arachnid) [inline EIPS/eip-ens-initial-registrar.md]: You can set this to `final`.

File: EIPS/eip-ens-initial-registrar.md [hunk 3]
```diff
@@ -0,0 +1,244 @@
...
+
+This restriction is enforced by the owner of the ENS root node who should not assign any nodes other than `.eth` to the Initial Registrar. The ENS's root node should be controlled by multiple parties using a multisig contract.
+
+The Initial Registrar will also prohibit registration of names 6 characters or less in length.
+
+### Name format for hash registration
+
+Names submitted to the initial registrar must be hashed using Ethereum's sha3 function. Note that the hashes submitted to the registrar are the hash of the subdomain label being registered, not the namehash as defined in EIP 137.
+
+For example, in order to register `abcdefg.eth`, one should submit `sha3('abcdefg')`, not `sha3('abcdefg', sha3('eth', 0))`.
```
Comment (Arachnid) [inline EIPS/eip-ens-initial-registrar.md]: The latter counterexample should be `sha3(sha3(0, 'eth'), 'abcdefg')`.

**Comment by maurelian** [comment]
@Arachnid I made the requested changes.

File: EIPS/eip-ens-initial-registrar.md [hunk 4]
```diff
@@ -0,0 +1,244 @@
...
+
+The following table outlines what portion of the balance held in a deed contract will be returned upon closure, and to whom. The remaining balance will be burnt.
+
+#### Refund schedule
+
+| Reason for Deed closure | Refund Recipient | Refund Percentage |
+| --- | --- | --- |
+| A valid non-winning bid is revealed. | Bidder | 99.5% |
+| A bid submitted after the auction period is revealed. | Bidder | 99.5% |
+| An otherwise valid bid is revealed on an owned name. <sup>1</sup> | Bidder | %0.5 |
```
Comment (ethers) [inline EIPS/eip-ens-initial-registrar.md]: Suggestion: A bid is revealed after the reveal period.   And fix to 0.5%

File: EIPS/eip-ens-initial-registrar.md [hunk 5]
```diff
@@ -0,0 +1,244 @@
...
+| An expired sealed bid is cancelled. <sup>2</sup> | Canceler | 0.5% |
+| A registered hash is reported as invalid. <sup>3</sup> | Reporter | 50% |
+| A registered hash is reported as invalid. <sup>3</sup> | Owner | 50% |
+
+##### Notes:
+
+1. This is to prevent an extortion attack on the current highest bidder, by threatening to reveal a new second highest bid. This forces all bids to be revealed.
```
Comment (ethers) [inline EIPS/eip-ens-initial-registrar.md]: Suggestion:  This incentivizes all bids to be revealed in time.  If bids could be revealed late, an extortion attack on the current highest bidder could be made by threatening to reveal a new second highest bid.

File: EIPS/eip-ens-initial-registrar.md (hunk 4 above)
Comment (maurelian) [inline EIPS/eip-ens-initial-registrar.md]: Sorry for not addressing this review sooner.  I think the actual implementation matches the current wording better. 

https://github.com/ethereum/ens/blob/mainnet/contracts/HashRegistrarSimplified.sol#L379

File: EIPS/eip-ens-initial-registrar.md (hunk 5 above)
Comment (maurelian) [inline EIPS/eip-ens-initial-registrar.md]: Good feedback!

**Comment by maurelian** [comment]
@Arachnid please close this. I see the suggested changes have been incorporated.

**Comment by Arachnid** [comment]
@maurelian It's already merged; it doesn't get more closed than that. :)

**Comment by maurelian** [comment]
ahh... yes. The open loop was just the one in my mind. :)

**Pull request merged**